### PR TITLE
Update default runtime-images to latest versions

### PR DESCRIPTION
**Pull request by lresende**
Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

File: etc/config/metadata/runtime-images/pytorch-devel.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
-  "display_name": "Pytorch 1.2 with CUDA-devel",
+  "display_name": "Pytorch 1.5 with CUDA-devel",
   "metadata": {
-    "image_name": "pytorch/pytorch:1.2-cuda10.0-cudnn7-devel"
+    "image_name": "pytorch/pytorch:1.5-cuda10.1-cudnn7-devel"
```
Comment (akchinSTC) [inline etc/config/metadata/runtime-images/pytorch-devel.json:4]: no curl in this image.suggest pinning to pytorch 1.4 instead...it does have it

File: etc/config/metadata/runtime-images/pytorch-runtime.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
-  "display_name": "Pytorch 1.2 with CUDA-runtime",
+  "display_name": "Pytorch 1.5 with CUDA-runtime",
   "metadata": {
-    "image_name": "pytorch/pytorch:1.2-cuda10.0-cudnn7-runtime"
+    "image_name": "pytorch/pytorch:1.5-cuda10.1-cudnn7-runtime"
```
Comment (akchinSTC) [inline etc/config/metadata/runtime-images/pytorch-runtime.json:4]: same with this one

**Comment by kevin-bates** [comment]
@akchinSTC - is `curl` our only prerequisite for images?   I'm assuming you checked the others as well (thanks).

**Comment by akchinSTC** [comment]
@kevin-bates yes just curl and python3. I checked the other images

**Comment by akchinSTC** [comment]
> Looks good. Thanks.
> 
> It might make sense to have script or something that validates images - ensuring they have the various commands. Something like:
> `docker run -it --rm <image> which curl`
> or mount a directory that contains a general validation script to run.

this would be a good test to run on all `image_name` under etc/config/metadata/runtime-images/*.json

**Pull request merged**